# Start scene info
# Attached collision objects (attached to some link on the robot)
moveit_msgs/AttachedCollisionObject[] attached_collision_objects

# collision objects
moveit_msgs/CollisionObject[] collision_objects